tools/tertiary-analysis/cell-types-analysis/ct_get_empirical_dist.xml

```diff
@@ -31,12 +31,14 @@
     <inputs>
         <param type="data" name="input_ref_file" label="Input reference file" format="txt" help="Text file with reference cell labels" />
         <param type="data" name="exclusions" format="yml" label="Exclusions file" optional="true" help="YML file with excluded/unlabelled terms" />
-        <param type="text" name="label_col_ref" label="Reference label column" value="cell_type" help="Label column in reference file" />
-        <param type="text" name="tmpdir" label="Tmpdir for caching" optional="true" help="Temporary directory for caching" />
+        <param type="text" name="label_col_ref" label="Reference label column" value="cell_type" help="Label column in reference file">
+        <expand macro="sanitize_strings" />
+        </param>
+        <param type="text" name="tmpdir" label="Tmpdir for caching" value="./" optional="true" help="Temporary directory for caching" />
```

Comment:
Same here, this shouldn't be an option for the user, please simply use "./" in the command section.